components/ledger/ledger-utxo-flow/src/main/kotlin/net/corda/ledger/utxo/flow/impl/flows/backchain/TransactionBackchainResolutionFlow.kt

```diff
@@ -4,6 +4,7 @@ import net.corda.flow.application.services.VersioningService
 import net.corda.flow.application.versioning.VersionedReceiveFlowFactory
 import net.corda.ledger.utxo.flow.impl.flows.backchain.v1.TransactionBackchainResolutionFlowV1
 import net.corda.ledger.utxo.flow.impl.flows.backchain.v2.TransactionBackchainResolutionFlowV2
+import net.corda.libs.platform.PlatformVersion.CORDA_5_1
```

Comment:
```suggestion
import net.corda.libs.platform.PlatformVersion.CORDA_5_1 as CURRENT_VERSION
```
Then it should be possible to use `CURRENT_VERSION` further in the file. This will help to change fewer lines in the next release when we will be introducing `CORDA_5_2`.

Reply:
@vkolomeyko I do not fully get this one. In this case, we want to introduce this new feature in a new flow from precisely 5.1.
Later versions (5.2) may or may not want to upgrade the flow, but the cutoff between v1 and v2 will be 5.1.

Reply:
Sorry, I misread what `TransactionBackchainResolutionFlow` is meant to do.
Specifying concrete versions makes sense to me here.